app/src/main/assets/html/Licenses.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
```

Comment:
Can you make a JIRA to create the real License for mPower Android (not iOS)? Don't want to ship with wrong license

Reply:
Done!